### ISSUE TITLE
An error in topic's "average time to answer" statistic - single user's results

### ISSUE THREAD

**Issue by tsfstarr** [tecnickcom/tcexam#351]
Hello, happy holidays.

I think I've spotted an erroneous "average time to answer". This is the Time column under Statistics in Test Results Summary, "Tests > Results" page.
The page path is /tcexam/admin/code/tce_show_result_allusers.php
I'm selecting an exam, a particular user, and "statistics > topic". Screenshot attached.

<img width="215" alt="tcexam_avg_time_error" src="https://user-images.githubusercontent.com/26569749/103090416-df89a080-45e8-11eb-839e-3df671dedcf6.png">

The spreadsheet below contains the times to answer the topic questions on this exam attempt, which seem to average about 3 minutes. However the average time displayed by TCExam for the topic is 9 minutes 49 seconds.
Just to check it wasn't an average across users, I selected no user, and the average for this topic across all users was different (14 minutes).

[tcexam_avg_time_error.xlsx](https://github.com/tecnickcom/tcexam/files/5739658/tcexam_avg_time_error.xlsx)

Kind regards

**Comment by tsfstarr** [comment]
I checked the other 6 topics for their average times, and vs my calculations they're all correct within 1 second.
It does make this particular topic average look erroneous.

Perhaps related...
I noticed that question timings on the Test Results Summary page (Tests > Results), and the Test Results page (Tests > Users), do not match for a given user's attempt.

Adding up all the 'average' question timings on Test Results Summary (for one user) gives a result larger than the duration of the attempt.
Adding up all the question timings on Test Results page gives roughly the duration of the attempt.
This makes the summary very confusing. I'm expecting an average of one question time measurement, given that this particular user has one attempt.
Perhaps it's simply that I don't understand what the Test Results Summary - Statistics "time" column is meant to represent.
